### ISSUE TITLE
Styling isn't removed after unlinking in conversion docs example

### ISSUE THREAD

**Issue by FilipTokarski** [ckeditor/ckeditor5#6385]
## 📝 Provide detailed reproduction steps (if any)

1. Go to https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/conversion/conversion-extending-output.html#adding-a-css-class-to-inline-elements
2. Scroll down to editor with green links
3. Unlink a link

I know it's just an example to give an idea about what the code does, but still, this can be a bit confusing for a user that sees it for the first time and tries to learn from this

### ✔️ Expected result

Text it unlinked, styles removed

### ❌ Actual result

![docs1](https://user-images.githubusercontent.com/34380544/76086815-286a5d00-5fb5-11ea-877f-c232fff39349.gif)

**Comment by Reinmar** [comment]
This is certainly an important issue. No idea why it's broken, but it'd be good to know whether it worked at any stage. Could you verify this and ping me?

**Comment by FilipTokarski** [comment]
@Reinmar it seems to be broken since it was introduced ( in v15 )

**Comment by FilipTokarski** [comment]
Actually right now when following the guide I get:
```js
```

**Comment by FilipTokarski** [comment]
Actually, right now when following the guide and trying to add a link, I get:
```js
Uncaught TypeError: item.callback is not a function
    at DowncastDispatcher.dispatcher.on.priority (automaticdecorators.js:77)
    at DowncastDispatcher.fire (emittermixin.js:199)
    at DowncastDispatcher._testAndFire (downcastdispatcher.js:564)
    at DowncastDispatcher.convertAttribute (downcastdispatcher.js:249)
    at DowncastDispatcher.convertChanges (downcastdispatcher.js:161)
    at editingcontroller.js:94
    at View.change (view.js:484)
    at Document.EditingController.listenTo.priority (editingcontroller.js:93)
    at Document.fire (emittermixin.js:199)
    at Document._handleChangeBlock (document.js:309)
```

**Comment by pomek** [comment]
Removing a link in the second editor also throws:

![image](https://user-images.githubusercontent.com/2270764/125457583-08531c47-93ad-4aa1-af72-99bcc028f996.png)

```
Uncaught TypeError: Cannot read property 'match' of null
    at u.e.on.priority (snippet.js:4)
    at u.fire (snippet.js:4)
    at u._testAndFire (snippet.js:4)
    at u.convertAttribute (snippet.js:4)
    at u.convertChanges (snippet.js:4)
    at snippet.js:4
    at U.change (snippet.js:4)
    at H.Y.listenTo.priority (snippet.js:4)
    at H.fire (snippet.js:4)
    at H._handleChangeBlock (snippet.js:4)
```

**Comment by AnnaTomanek** [comment]
This example was removed in the new conversion docs so we can close it as invalid. Or closed by #10911 :smile:
